README.md

```diff
@@ -415,6 +415,14 @@ FFMPEGKIT_PACKAGE_VERSION=5.1 npx pod-install ios
 FFMPEGKIT_PACKAGE=full FFMPEGKIT_PACKAGE_VERSION=5.1 npx pod-install ios
 ```
 
+If you use anything other than `https` ensure you have this added to your pod spec
```

Comment:
this should not be in customize FFMPEG package section. The section is about this lib only, not related `ffmpeg-kit-react-native`

Whatever related to `ffmpeg-kit-react-native` is already mentioned here [Naming conflict with ffmpeg-kit-react-native](https://github.com/maitrungduc1410/react-native-video-trim?tab=readme-ov-file#naming-conflict-with-ffmpeg-kit-react-native)

Reply:
Cook I get you